protos/test_run.proto

```diff
@@ -16,20 +16,26 @@ syntax = "proto3";
 
 package wptdashboard;
 
+import "bq_table_name.proto";
+import "browser.proto";
+import "engine.proto";
 import "google/protobuf/timestamp.proto";
+import "operating_system.proto";
 
+// An invocation of the WPT runner in some OS+browser context.
 message TestRun {
-  google.protobuf.Timestamp created_at = 1;
+  option (gen_bq_schema.table_name) = "test_runs";
 
-  // The first 10 characters of the SHA1 of the tested WPT revision
-  string revision = 2;
+  google.protobuf.Timestamp start_time = 1;
+  google.protobuf.Timestamp end_time = 2;
```

Comment:
These fields are prefixed with `test_run_*` in `TestResult`

Reply:
Yes. The intent there is to qualify that it is the "test run's start/end time" in the context of a TestResult; in the context of a TestRun, that seems verbose, IMHO.

Reply:
Ok, I see now and agree with your thinking—thanks for clarifying